src/components/MyKiva/JournalUpdatesCarousel.vue

```diff
@@ -187,16 +188,16 @@ const loadMoreUpdates = () => {
 
 watch(
 	() => updates,
-	() => {
-		if (updates.value.length) {
+	newUpdates => {
+		if (newUpdates.value.length) {
 			$kvTrackEvent('portfolio', 'view', 'At least one journal update viewed');
 		}
-		if (updates.value.length > 0 && updates.value.length < 3) {
-			carouselIndex.value = 0;
-		}
-		if (updates.value.length > 3) {
-			carouselIndex.value = updates.value.length - 2;
+
+		if (prevUpdatesLength > 0 && prevUpdatesLength !== newUpdates.value?.length) {
+			carouselIndex.value = prevUpdatesLength - 1;
```

Comment:
It looks like this should be `carouselIndex.value = prevUpdatesLength` to move to the first new item, correct?

Reply:
I think maybe moving to the first new item might be clearer.